src/Mapping.elm

```diff
@@ -0,0 +1,386 @@
+module Mapping exposing (..)
+
+--Exercise 1: Double the Numbers
+--Task: Write a function that doubles each number in a list of integers.
+
+
+double : Int -> Int
+double integer =
+    integer * 2
+
+
+doubleIntegers : List Int -> List Int
+doubleIntegers integers =
+    List.map double integers
+
+
+
+--Exercise 2: Convert Boolean to String
+--Task: Convert each boolean in a list to its string representation ("true" or "false").
+
+
+boolToString : Bool -> String
+boolToString bool =
+    if bool then
+        "True"
+
+    else
+        "False"
+
+
+stringFromBool : List Bool -> List String
+stringFromBool listBool =
+    List.map boolToString listBool
+
+
+
+--Exercise 3: Calculate Areas of Circles
+--Task: Given a list of circle radii, calculate the area for each corresponding circle.
+
+
+areaOfCircle : Float -> Float
+areaOfCircle radius =
+    let
+        pi =
+            3.14159
+    in
+    pi * radius * radius
+
+
+areaOfCircles : List Float -> List Float
+areaOfCircles radii =
+    List.map areaOfCircle radii
+
+
+
+--Exercise 4: Calculate Volumes of Cubes/Cuboids
+--Task: Given a list of records with dimensions of type Dimensions,
+--calculate the volume for each cuboid.
+
+
+type alias Dimensions =
+    { length : Float
+    , height : Float
+    , breadth : Float
+    }
+
+
+volume : Dimensions -> Float
+volume dimension =
+    dimension.length * dimension.height * dimension.breadth
+
+
+
+-- Calculate volumes for each dimension using the map function
+
+
+volumes : List Dimensions -> List Float
+volumes dimensions =
+    List.map volume dimensions
+
+
+
+--Exercise 5: Uppercase Strings
+--Task: Convert all strings in a list to uppercase.
+
+
+stringsToUpper : List String -> List String
+stringsToUpper strings =
+    List.map String.toUpper strings
+
+
+
+--Exercise 6: Calculate Square Roots
+--Task: Return the square roots of a list of numbers.
+
+
+squareRoot : Float -> Float
+squareRoot float =
+    sqrt float
```

Comment:
Nah. You don't need to define a `squareRoot`. You can just use Elm's `sqrt` directly inside `squareRoots`.